08-components-lab/src/02-numberguessing.jsx

```diff
@@ -13,5 +13,45 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class NumberGuessing extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            origNumber: Math.floor((Math.random() *1000)+1),
+            number:0
+        }
+    }
+
+    calc = (evt) => {
+        this.setState({number:Number(evt.target.value)});
+
+    }
+
+    getStatus(){
+        //random number between 1-10
+        let real = Math.floor((Math.random() *10)+1);
+        console.log("If real smaller then 8, compare original with number, otherwise, just print Too Big or Too Small");
+        console.log("Real: "+real);
+
+        if (this.state.origNumber === this.state.number)
```

Comment:
Instead of printing randomly "Too small" or "Too big" when hitting random number above 8, Make sure that for some cases pc is lying. It means that sometimes it will "definitely lie" and show the opposite.